mautic/mautic
# Tags can be imported via csv

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | https://github.com/mautic/user-documentation/pull/98/files#r1280294088
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds possibility to import contacts with tags from a CSV


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Import contacts with tags. The mock file can be used:  [MOCK_DATA_3_CONTACT.csv](https://github.com/mautic/mautic/files/11983946/MOCK_DATA_3_CONTACT.csv)
![image](https://github.com/mautic/mautic/assets/96085911/f3e2fde5-22e9-4476-a435-2903930d491c)

3. Checking that tags are added to contacts
